demo/ReadText.LocalizedDemo/Program.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using CommandLine;
+using CommandLine.Text;
+
+namespace ReadText.LocalizedDemo
+{
+    class Program
+    {
+        public static int Main(string[] args)
+        {
+            // Set sentence builder to localizable
+            SentenceBuilder.Factory = () => new LocalizableSentenceBuilder();
+
```

Comment:
//Apply localization for 'cs'
            var culture = CultureInfo.CreateSpecificCulture("cs");
            Thread.CurrentThread.CurrentCulture = culture;
            Thread.CurrentThread.CurrentUICulture = culture;